gems/aws-sdk-core/spec/aws/plugins/retry_errors_spec.rb

```diff
@@ -1,9 +1,9 @@
 require_relative '../../spec_helper'
+require_relative '../../support/retry_errors_helper'
```

Comment:
We can add tests here that validate given a config, the appropriate handler is loaded